indra/resources/default_config.ini

```diff
@@ -12,6 +12,9 @@ EIDOSPATH =
 # Path to the Sparser reading system folder
 SPARSERPATH = 
 
+# Path to TEES
+TEES_PATH = ~/TEES
```

Comment:
I'd remove your specific local path from the default config and just leave it blank